client/src/pages/LandingPage/useStyles.ts

```diff
@@ -0,0 +1,27 @@
+import { makeStyles } from '@mui/styles';
+import dogHero from '../../images/landing/landing_page.jpg';
+import { theme } from '../../themes/theme';
+
+const useStyles = makeStyles(() => ({
+  rightHalf: {
+    backgroundImage: `url(${dogHero})`,
+    backgroundRepeat: 'no-repeat',
+    backgroundPosition: 'center',
+    objectFit: 'contain',
```

Comment:
```suggestion
    objectFit: 'cover',
```
I would suggest using `cover` because on larger screens when the image is contained we may have white space around the image